neurodocker/interfaces/afni.py

```diff
@@ -161,3 +186,62 @@ def install_binaries(self):
         env_cmd = indent("ENV", env_cmd)
 
         return "\n".join((env_cmd, cmd))
+
+    def install_from_source(self):
+        """Return Dockerfile instructions to download and build AFNI from
+        source."""
+        pkgs = self._get_source_dependencies()
+
+        cmd = ("{install}"
+               "".format(**manage_pkgs[self.pkg_manager]).format(pkgs=pkgs))
+
+        if self.pkg_manager == "yum":
+            cmd += ("\n&& yum install -y -q R R-devel")
```

Comment:
Can `R` and `R-devel` be added to the list of `yum` dependencies in `_get_source_dependencies()`?

Reply:
The `R` and `R-devel` packages are found in the EPEL repository.  The `epel-release` package is installed in the yum source dependency list, and therefore both R packages need to come after in a separate `yum` invocation.